CSS/style.css

```diff
@@ -101,7 +104,15 @@ form li {
 
 .home p {
   width: 270px;
-  font-size: 16px;
+  font-size: 20px;
+}
+
+/* .connect p {
+  color: #7f8cff;
+} */
```

Comment:
Please do not leave commented code in your codebase. It makes your code look messy and violates the DRY principle of coding.🙏